neo/test/rawiotest/test_neuralynxrawio.py

```diff
@@ -86,5 +158,137 @@ def test_recording_types(self):
             self.assertEqual(hdr.type_of_recording(), typeTest[1])
 
 
+class TestNcsBlocksFactory(TestNeuralynxRawIO, unittest.TestCase):
+    """
+    Test building NcsBlocks for files of different revisions.
+    """
+
+    def test_ncsblocks_partial(self):
+        filename = self.get_filename_path('Cheetah_v6.3.2/incomplete_blocks/CSC1_reduced.ncs')
+        data0 = np.memmap(filename, dtype=NeuralynxRawIO._ncs_dtype, mode='r',
+                          offset=NlxHeader.HEADER_SIZE)
+        self.assertEqual(data0.shape[0], 6690)
+        self.assertEqual(data0['timestamp'][6689], 8515800549)  # timestamp of last record
+
+        hdr = NlxHeader.build_for_file(filename)
+        nb = NcsBlocksFactory.buildForNcsFile(data0, hdr)
+        self.assertEqual(nb.sampFreqUsed, 32000.012813673042)
+        self.assertEqual(nb.microsPerSampUsed, 31.249987486652431)
+        self.assertListEqual([blk.startBlock for blk in nb.blocks], [0, 1190, 4937])
+        self.assertListEqual([blk.endBlock for blk in nb.blocks], [1189, 4936, 6689])
+
+    def testBuildGivenActualFrequency(self):
```

Comment:
Please, also for test methods, follow the PEP8 standard and use lower case names and underscores instead of mixed case names, see also https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

Reply:
Done.